torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -735,6 +736,19 @@ def _post_backward_hook(
     - Otherwise, the ``_saved_grad_shard`` attribute is the reduced sharded
     gradient (accumulating with any existing gradient).
     """
+    # Under TORCH_DISTRIBUTED_DEBUG=INFO, log the module names this hook fires for.
```

Comment:
nit: IMHO, only the part that explains when we expect hooks to not fire (FSDP + AC) is important info the document. The other info is all clear from the code (e.g. line 742):
```
if state._use_orig_params and dist.get_debug_level() == dist.DebugLevel.INFO:
```